src/UCTSearch.cpp

```diff
@@ -224,8 +225,16 @@ SearchResult UCTSearch::play_simulation(GameState & currstate,
         }
     }
 
+    auto depth =
+        int(currstate.get_movenum() - m_rootstate.get_movenum());
+    auto apparent_depth = depth;
+    if (is_pondering_now == true) { // Adjust depth by 1 to account for thinking/pondering during opponent's turn
+        apparent_depth =
```

Comment:
Thanks for taking the time to look this over.

One issue with remembering computer color is that sometimes players "switch places" with the computer during a game for various reasons, so it might be best to set/update this whenever LZ enters pondering.

I will submit a more elegant handling of this tomorrow.

Reply:
Are you still working on this?

Reply:
Yes, incoming very shortly thanks to this reminder.

Reply:
Rewrite nearly finished, along with a few questions about best practices when I post it